src/tests/e2e/login.test.e2e.js

```diff
@@ -18,4 +27,43 @@ describe('Login and onboarding', () => {
     },
     timeout
   )
+
+  it(
+    'Login as a new user',
+    async () => {
+      // close non in
+      await page.close()
+      // first login to disconnect page to delete cloud wallet
+      let context = await browser.createIncognitoBrowserContext()
+      // Create a new page inside context.
+      page = await context.newPage()
+
+      await page.goto(`${process.env.E2E_TEST_URL}/disconnect`)
+      const loginPage = new LoginPage()
+      await loginPage.login(
+        process.env.E2E_TEST_GOOGLE_LOGIN_USERNAME,
+        process.env.E2E_TEST_GOOGLE_LOGIN_PASSWORD
+      )
+      const disconnectPage = new DisconnectPage()
+      await disconnectPage.disconnect()
+
+      // reset page and close incognito page
+      await jestPuppeteer.resetPage()
+        
+      context = await browser.createIncognitoBrowserContext()
+      // Create a new page inside context.
+      page = await context.newPage()
+      await page.goto(`${process.env.E2E_TEST_URL}`)
+      // now test account is a new user
+      await loginPage.login(
+        process.env.E2E_TEST_GOOGLE_LOGIN_USERNAME,
+        process.env.E2E_TEST_GOOGLE_LOGIN_PASSWORD
+      )
+
+      // once fully login, check if there is any error snakbar
+      // wait for 5 seconds
+      await sleep(5000)
```

Comment:
no need to sleep, loginPage.login already waits till all network calls finish

Reply:
fixed